### PR TITLE
fix(p-search): fix contextMenu position bug

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [x] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console(if usages are changed) 

### Description
before
image

after
image

### CODE REVIEW

File: src/inputs/search/search/PSearch.vue [hunk 1]
```diff
@@ -152,29 +152,29 @@ export default defineComponent<SearchProps>({
...
             searchableItems: computed<MenuItem[]>(() => props.menu.filter((d) => d.type === undefined || d.type === 'item')),
             bindingMenu: computed<SearchDropdownMenuItem[]>(() => (props.disableHandler ? props.menu : state.filteredMenu)),
             proxyValue: useProxyValue('value', props, emit),
             menuRef: null,
         });
+        const {
+            targetRef, targetElement, contextMenuStyle,
+        } = useContextMenuFixedStyle({
+            useFixedMenuStyle: computed(() => props.useFixedMenuStyle),
+            visibleMenu: toRef(state, 'proxyVisibleMenu'),
```
Comment (minseolee) [inline src/inputs/search/search/PSearch.vue:173]: 😃

File: src/inputs/search/search/PSearch.vue [hunk 2]
```diff
@@ -211,11 +211,11 @@ export default defineComponent<SearchProps>({
         }, 300);
 
         const showMenu = () => {
-            if (state.isAutoMode) contextMenuFixedStyleState.proxyVisibleMenu = true;
+            state.proxyVisibleMenu = true;
             emit('show-menu');
         };
         const hideMenu = () => {
-            if (state.isAutoMode) contextMenuFixedStyleState.proxyVisibleMenu = false;
```
Comment (minseolee) [inline src/inputs/search/search/PSearch.vue:218]: Has `isAutoMode` deprecated?

**Review by minseolee — APPROVED** [review]
Nice~! Left small comments

File: src/inputs/search/search/PSearch.vue (hunk 2 above)
Comment (sulmoJ) [reply src/inputs/search/search/PSearch.vue:218]: It was a feature that had not existed before, but it was added incorrectly during the previous implementation. 😭

File: src/inputs/search/search/PSearch.vue (hunk 1 above)
Comment (WANZARGEN) [inline src/inputs/search/search/PSearch.vue:173]: praise: oh... You found out..!!!!! Greeaaaaaaat 👍

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**